src/EA.Weee.Web/ViewModels/Shared/EvidenceNoteViewModel.cs

```diff
@@ -47,10 +47,15 @@ public class EvidenceNoteViewModel
 
         public int Reference { get; set; }
 
-        [DisplayName("Reason")]
-        public string Reason { get; set; }
+        [DisplayName("Reason returned")]
+        public string ReturnedReason { get; set; }
 
```

Comment:
Not really this changes issues, but could we add a test to check that the ReturnedReason and RejectedReason properties have a DisplayName attribute with the correct value. 

e.g. typeof(EditEvidenceNoteViewModel).GetProperty(property).Should()
                .BeDecoratedWith<DisplayNameAttribute>(a => a.DisplayName.Equals(display));